### PR TITLE
Add a 'hidden' option for labs

### PR DESCRIPTION
**Pull request by mrtmm**
Add an option to hide the lab window while spinning up a
lab environment in the background.

### CODE REVIEW

**Review by fghaas — COMMENTED** [review]
Maybe I'm missing something, but I don't quite see how this ensures that a `hidden` XBlock is also treated as read-only. Could you explain how that works?

**Comment by mrtmm** [comment]
I forgot to `git add` that line, included now.

**Comment by mrtmm** [comment]
Good point, thanks.

**Review by fghaas — COMMENTED** [review]
One more micro-nit and then this is good to merge.

File: Changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Unreleased
+-------------------------
+* [Enhancement] Add `hidden` option for spinning up a lab
+  environment on the background while the lab itself is hidden.
```
[the remark this answers is not in the document]
Comment (mrtmm) [reply Changelog.md]: Ack, done.

**Review by fghaas — APPROVED** [review]
This looks good to me now, thank you!

**Pull request merged**